scripts/apps/search/services/TagService.ts

```diff
@@ -168,24 +188,37 @@ export function TagService($location, desks, userList, metadata, search,
     var fieldProcessors = {
         original_creator: (index, value) => {
             userList.getUser(index).then((user) => {
-                tags.selectedParameters.push(tag(value + ':' + user.display_name));
+                const tagValue = value + ':' + user.display_name;
+
+                tags.selectedParameters.push(tag(tagValue, tagValue));
             }, (error) => {
-                tags.selectedParameters.push(tag(value + ':Unknown'));
+                const tagValue = value + ':Unknown';
```

Comment:
I guess we should eventually translate that `Unknown`

Reply:
I've translated it in https://github.com/superdesk/superdesk-client-core/pull/3593